test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -380,19 +380,32 @@ public void TensorFlowTransformObjectDetectionTest()
         [Fact(Skip = "Model files are not available yet")]
         public void TensorFlowTransformInceptionTest()
         {
-            var modelLocation = @"C:\models\TensorFlow\tensorflow_inception_graph.pb";
+            string inputName = "input";
+            string outputName = "softmax2_pre_activation";
+            var modelLocation = @"inception5h\tensorflow_inception_graph.pb";
```

Comment:
>inception5h [](start = 34, length = 11)

Has this model been uploaded to Microsoft.ML.TensorFlow.TestModels? At the time that this test was written it was skipped because we needed to handle the license issue.

Reply:
I think we haven't sorted out license issue yet. Have we?

Reply:
Otherwise, we can download the model on the fly but it will take time during tests runs.

Reply:
I will do it as part of [https://github.com/dotnet/machinelearning/issues/904](https://github.com/dotnet/machinelearning/issues/904) in separate PR.

---
In reply to: [262729625](https://github.com/dotnet/machinelearning/pull/2855#discussion_r262729625) [](ancestors = 262729625)